ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17__periodics-mce.yaml

```
@@ -0,0 +1,156 @@
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-aws-conformance
+- as: e2e-agent-connected-ovn-ipv4-manual-conformance
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-manual-conformance
+- as: e2e-agent-connected-ovn-ipv4-metal3-conformance
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-metal3-conformance
+- as: e2e-agent-connected-ovn-dualstack-metal3-conformance
+  cron: 0 8 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=500G
+        NUM_EXTRA_WORKERS=3
+      IP_STACK: v4v6
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-metal3-conformance
+- as: e2e-agent-connected-ovn-ipv4-metal3-compact-conformance
+  cron: 0 12 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NUM_WORKERS=0
+        NUM_MASTERS=3
+        MASTER_VCPU=16
+        MASTER_MEMORY=40960
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=500G
+        NUM_EXTRA_WORKERS=3
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-metal3-conformance
+- as: e2e-agent-disconnected-ovn-ipv6-metal3-conformance
+  cron: 0 16 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-disconnected-conformance
+- as: e2e-agent-disconnected-ovn-dualstack-metal3-conformance
+  cron: 0 20 * * *
+  steps:
+    cluster_profile: equinix-ocp-hcp
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4v6
+        MIRROR_IMAGES=true
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=500G
+        NUM_EXTRA_WORKERS=3
+      IP_STACK: v4v6
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-agent-disconnected-conformance
+- as: e2e-kubevirt-metal-ovn
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      HYPERSHIFT_NODE_COUNT: "2"
+      LVM_OPERATOR_SUB_CHANNEL: stable-4.17
+      MCE_VERSION: "2.7"
+      ODF_OPERATOR_SUB_CHANNEL: stable-4.16
+    workflow: hypershift-mce-kubevirt-baremetalds-conformance
+- as: e2e-ibmz-ovn-conformance
+  cron: 0 4 * * *
+  steps:
+    cluster_profile: aws
+    env:
+      MCE_VERSION: "2.7"
+    workflow: hypershift-mce-ibmz-conformance
+- as: e2e-power-ovn-conformance
+  cron: 0 6 * * *
```

We want runs to be apart for monitoring purpose.
```suggestion
  cron: 0 8 * * *
```